src/posts/posts.json

```
@@ -1072,5 +1072,15 @@
     "authors": ["michael-smit"],
     "filename": "api-v2.md",
     "image": "api-v2.webp"
+  },
+  {
+    "title": "The New York City Child Tax Credit",
+    "date": "2025-02-18 00:00:01",
+    "authors": ["David Trimmer"],
+    "tags": ["NYC", "child tax credit", "policy"],
```

us to go to the right country homepage, other tags please check against existing tags
```suggestion
    "tags": ["us", "us-ny-nyc", "child tax credit", "policy"],
```